task/01-strings-tasks.js

```diff
@@ -221,7 +281,35 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+  if (!str) return str
```

Comment:
Well done! But this code looks a bit messy and potentially it could be difficult to read by other developers. Can you try to come up with another, shorter and more readable solution?

Hint: use `replace`, `String.fromCharCode`, and `charCodeAt` methods